packages/account/src/Components/sent-email-modal/sent-email-modal.jsx

```diff
@@ -18,6 +18,11 @@ const getNoEmailContentStrings = () => {
                 'You accidentally gave us another email address (Usually a work or a personal one instead of the one you meant).'
             ),
         },
+        {
+            key: 'wrong_typo',
+            icon: 'IcEmailTypo',
+            content: localize('The email address you entered had a mistake or typo (happens to thr best of us).'),
```

Comment:
Typo - happens to the best of us 🤣 
```suggestion
            content: localize('The email address you entered had a mistake or typo (happens to the best of us).'),
```